ci/citest/acceptance-test/tests/cmd_console_test.go

```diff
@@ -22,11 +22,12 @@ func runConsoleCmd(instance_id string, t *testing.T) {
 	RunCmdAndExpectFail(t, "sh", "-c", fmt.Sprintf("openvdc console %s -- false", instance_id))
 }
 
-func runConsoleCmdWithPrivatekey(instance_id string, private_key_path string, t *testing.T) {
-	RunCmdAndReportFail(t, "openvdc", "console", instance_id, "-i", private_key_path)
-	RunCmdAndReportFail(t, "sh", "-c", fmt.Sprintf("openvdc console %s -i %s ls", instance_id, private_key_path))
-	RunCmdAndReportFail(t, "sh", "-c", fmt.Sprintf("openvdc console %s -i %s -- ls", instance_id, private_key_path))
-	RunCmdAndExpectFail(t, "sh", "-c", fmt.Sprintf("openvdc console %s -i %s -- false", instance_id, private_key_path))
+func runConsoleCmdWithPrivatekey(instance_id string, private_key_path string, t *testing.T, expect_fail bool) {
+	if expect_fail {
+		RunCmdAndExpectFail(t, "openvdc", "console", instance_id, "-i", private_key_path)
+	} else {
+		RunCmdAndReportFail(t, "openvdc", "console", instance_id, "-i", private_key_path)
+	}
```

Comment:
This function is only called twice. Once with expect_fail = true and once with expect_fail = false.

Since only one line gets executed for each call, I think it's better to write those lines directly in the test instead of in a separate function.

Reply:
Deleted runConsoleCmdWithPrivatekey and integrated to TestLXCCmdConsole_AuthenticationPubkey.